doc/source/examples/incompressible-flow/3d_nitsche_mixer_with_pbt_impeller/nitscher_mixer_with_pbt_impeller.rst

```diff
@@ -0,0 +1,279 @@
+
+The setup that we wish to simulate is schematized in the following figure:
+
+  .. image:: images/scheme_tank.png
+   :alt: The geometry and boundary conditions
+   :align: center
+   :name: geometry
+   :height: 12cm
+
+
+
+It consists of a PBT of diameter :math:`D` in a vessel of diameter :math:`T`. The vessel is at a clearance :math:`C` from the bottom. Without loss of generality, we will consider :math:`T=1` and make the problem dimensionless by considering all other dimensions are ratio of :math:`T`. The diameter of the impeller is :math:`D/3` and the clearance is :math:`D/4`. The Reynolds number of the flow will be controlled through the viscosity and the angular velocity of the impeller.
+
+The PBT is rotating at a constant velocity. On the lateral and bottom walls of the vessel, no-slip boundary conditions are applied. On the top surface, a slip boundary condition is applied to mimic the liquid's free surface. This choice of boundary condition is debatable, a do-nothing boundary condition could be used, since free surfaces have a constant pressure, but a slip boundary condition is generally significantly more robust numerically.
+
+For this configuration, the Reynolds number is defined as:
+
+  .. math::
+    Re = \frac{ND^2}{\nu}
+
+where :math:`N` is the angular velocity in Rotation Per Second (RPS) and :math:`\nu` is the kinematic viscosity. We investigate this case at a Reynolds number of 300 for the flow reaches a pseudo steady-state. However, this example can easily be extend to higher values of the Reynolds number by lowering the kinematic viscosity. It will be necessary to increase the mesh resolution for the solution to be meaningfully accurate.
+
+
+Parameter file
+--------------
+
+All the sections of the parameter file used in this case have been already explained in previous examples with the exception of the Nitsche immersed boundary. However, for the sake of completeness, the important sections are briefly explained.
+
+Simulation control
+~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+
+    subsection simulation control
+      set method                  = bdf2
+      set output name             = mixer
+      set output path             = ./output/
+      set output frequency        = 2
+      set time end                = 5
+      set time step               = 0.02
+      set output boundaries       = true
+    end
+
+Since the flow generated by the agitator is naturally transient, we use a second-order ``bdf2`` time-stepping method. We simulate the flow for a total of 5 seconds. We set ``output boundaries = true`` to generate auxiliary visualisation file that will enable us to visualise the walls of the vessel without the inside cells and  thus facilitate the visualisation of our mixing set-up.
```

Comment:
```suggestion
Since the flow generated by the agitator is naturally transient, we use a second-order ``bdf2`` time-stepping method. We simulate the flow for a total of 5 seconds. We set ``output boundaries = true`` to generate an auxiliary file that highlights the equipment walls by hiding the cells of the mesh. This allows for a clearer visualization of the mixing setup.
```